src/diffpy/labpdfproc/labpdfprocapp.py

```diff
@@ -55,15 +55,26 @@ def get_args():
         "-f",
         "--force-overwrite",
         action="store_true",
-        help="outputs will not overwrite existing files unless --force is spacified",
+        help="outputs will not overwrite existing file unless --force is specified",
```

Comment:
I think I have seen this fix on multiple branches (not sure) but only fix things in one branch or your generate conflicts (as we have been finding)